frontend/src/GlobalDatasources/GlobalDataSourcesPage/index.jsx

```diff
@@ -63,6 +64,7 @@ export const GlobalDataSourcesPage = ({ darkMode = false, updateSelectedDatasour
   }, []);
 
   useEffect(() => {
+    setWindowTitle({ page: `${selectedDataSource?.name || 'Data sources'}` });
```

Comment:
**Dev-guidelines Suggestion**
Shall we move these hard coded texts ("Page names") to constant in future.